capabilities/src/execute_task_solution_capability.cpp

```diff
@@ -193,26 +193,28 @@ bool ExecuteTaskSolutionCapability::constructMotionPlan(const moveit_task_constr
 	auto make_description = [size = solution.sub_trajectory.size()](const std::size_t index) {
 		return std::to_string(index + 1) + "/" + std::to_string(size);
 	};
-	// TODO: Case for only one subtrajectory
-	if (!make_executable_trajectory(solution.sub_trajectory[0], make_description(0),
-	                                make_apply_planning_scene_diff_cb({ solution.sub_trajectory[0].scene_diff,
-	                                                                    solution.sub_trajectory[1].scene_diff })))
-		return false;
-	for (size_t i = 1; i < solution.sub_trajectory.size() - 1; ++i) {
+
+	// always include initial scene diff
+	std::vector<moveit_msgs::msg::PlanningScene> scene_diffs = { solution.sub_trajectory[0].scene_diff };
```

Comment:
nit, but should we reserve the size of this variable since we know `solution.sub_trajectory.size()` up front? Other vectors earlier in this function do this as well

Reply:
done (even though I think the performance impact is negligible here)